qark/plugins/Manifest_File_Checks.py

```diff
@@ -58,66 +104,51 @@ def target(self, queue):
                 launch_mode_list.append(line)
 
             if any(re.findall(self.TASK_REPARENTING, line)):
-                PluginUtil.reportInfo(fileName, self.TaskReparentingIssue(fileName), res)
+                PluginUtil.reportInfo(file_name, task_reparenting(file_name), res)
+
+            if re.match(self.API_KEY_REGEX, line):
+                # Check if special character is present in the line. If "Yes, then ignore.
+                if re.match(self.SPECIAL_CHAR_REGEX, line):
+                    pass
+                else:
+                    if line not in api_key_list:
+                        api_key_list.append(line)
+
+            # Additional check for hardcoded api keys which matches the syntax most commonly used with google API_KEY
+            if any(re.findall(self.HARDCODED_API_KEY, line)):
```

Comment:
So `findall` will return a list of matches if it can. `any` will check that there is something in the list that it returns. Since you aren't actually using any of the matches that findall is returning and are only checking that it is finding something, you should use `re.search` or `re.match` instead.